src/scss/components/_legal-links.scss

```diff
@@ -1,29 +1,21 @@
-.legal-links {
-  position: absolute;
-  bottom: -3em;
-  left: 0;
-  width: 100%;
-  display: -webkit-box;
-  display: -ms-flexbox;
+#legal-links {
```

Comment:
I think it would be nice to keep these stuck to the bottom on mobile, so they don't jump around when the spinner appears for instance.

Here's an idea for what you could do, copied from my messing around in devtools:

```
/* Element | http://localhost:3000/ */

.card {
  min-height: 100%;
  display: flex;
  flex-direction: column;
}

/* Element | http://localhost:3000/ */

body {
  height: 100%;
}

/* Element | http://localhost:3000/ */

#nlx-form {
  flex-grow: 1;
}
```

If you take this route, you'll want to test it more than I did, and unset that `min-height` at the breakpoint, to ensure the card doesn't become very long on desktop.

Reply:
> and unset that min-height at the breakpoint, to ensure the card doesn't become very long on desktop

![Screen Shot 2020-05-01 at 17 09 08](https://user-images.githubusercontent.com/755354/80820464-d0c02a80-8bce-11ea-976f-0a4d54b461c9.png)

Reply:
Done! I set the height breakpoint at 824px, so it should stick to the bottom for people with an iPhone X* or Pixel XL.